src/app/pages/registration/components/ear-tag-icon/ear-tag-icon.component.scss

```diff
@@ -0,0 +1,60 @@
+:host {
+	height: auto;
+}
+
+.new-ear-tag {
+	width: auto;
+	height: auto;
+
+	.triangle {
+		width: 0;
+		height: 0;
+		border-style: solid;
+	}
+
+	.ear-tag-row {
+		display: flex;
+	}
+	
+	.top-left {
+		border-width: 0 0 10px 6px;
+		border-color: transparent transparent transparent transparent;
```

Comment:
Kan du ikke bare skrive én transparent her slik at det blir satt for alle sidene?